### PR TITLE
 Show information in folder panel if ignore patterns are active fixes #1279 

### LINKED ISSUE #1279: Show information in folder panel if ignore patterns are active
Currently there is no visual indication in the web GUI if ignore patterns are used in a folder.
Please add line "ignore patterns .......... yes" to folder panels, only if they are active (like "compression ..... no" or "folder master ..... yes"). optional: clicking on yes opens ignores editor modal.

### PR DESCRIPTION
**Pull request by krozycki**
Fixes #1279 (the obligatory part)

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
Can you rebase your commit on top of master so that we could get rid of the `Merged changes` commit?

Also, I am not sure if just specifying the `#1279` will work, I think Github checks for words `fix, fixed, close, closes`, so just make it `(fixes #1279)`.

Sorry for being so picky though :dancer:

**Comment by krozycki** [comment]
Done.
Sure, no problem, it's good that you're maintaining order here. :+1:

File: cmd/syncthing/gui.go [hunk 1]
```diff
@@ -302,6 +302,15 @@ func restGetModel(m *model.Model, w http.ResponseWriter, r *http.Request) {
 	res["state"], res["stateChanged"] = m.State(folder)
 	res["version"] = m.CurrentLocalVersion(folder) + m.RemoteLocalVersion(folder)
 
+	ignorePatterns, _, _ := m.GetIgnores(folder)
+	nonTrivialPattern := false
+	for _, line := range ignorePatterns {
+		if len(line) > 0 && !strings.HasPrefix(line, "//") {
+			nonTrivialPattern = true
```
Comment (AudriusButkevicius) [inline cmd/syncthing/gui.go]: I guess you could break out of iteration once this becomes true, as there is no point to continue the iteration.
Also, you could assign the value directly to `res["ignorePatterns"]`

**Comment by krozycki** [comment]
Done.

**Comment by AudriusButkevicius** [comment]
I'll take this for a spin in the evening :+1:

**Pull request merged**